documentation/library_guide/introduction.rst

```diff
@@ -98,6 +112,20 @@ Known Limitations
 * Due to specifics of Microsoft* Visual C++, some standard floating-point math functions
   (including ``std::ldexp``, ``std::frexp``, ``std::sqrt(std::complex<float>)``) require device support
   for double precision. 
+* The initial value type for ``exclusive_scan``, ``inclusive_scan``, ``exclusive_scan_by_segment``,
```

Comment:
My changes in bold:
* The initial value type for ``exclusive_scan``, ``inclusive_scan``, ``exclusive_scan_by_segment``,
  ``inclusive_scan_by_segment``, ``transform_exclusive_scan``, ``transform_inclusive_scan`` **satisfies** 
  the ``DefaultConstructible`` requirements. A default constructed-instance
  of the initial value type **becomes** the identity element for binary_op. 
* The initial value type for ``exclusive_scan``, ``inclusive_scan``, ``exclusive_scan_by_segment``,
  ``inclusive_scan_by_segment``, ``reduce``, ``reduce_by_segment``, ``transform_reduce``, ``transform_exclusive_scan``,
  ``transform_inclusive_scan`` **satisfies** the ``MoveAssignable`` and the ``CopyConstructible``
  requirements.
* For ``max_element``, ``min_element``, ``minmax_element``, ``partial_sort``,
  ``partial_sort_copy``, ``sort``, ``stable_sort`` the dereferenced value type of
  the provided iterators **is** ``DefaultConstructible``.
* For ``remove``, ``remove_if``, ``unique`` the dereferenced value type of the provided
  iterators **is** ``MoveConstructible``.

Reply:
Added the suggestion

Reply:
As far as I can judge, dropping "shall" here results in the loss of the meaning that these notes describe requirements to the user-provided input and its C++ types. I am going to restore it back.

Reply:
Please don't restore the shall instances. Shall is a very formal indicator and we try to have our content more readable for multiple audiences and Google translation.